<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/Images/lemon.png</normaloff>:/icons/Images/lemon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="treeView"/>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="listView"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_All"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menuFind_Replace">
     <property name="title">
      <string>Find/Replace</string>
     </property>
     <addaction name="actionQuick_Find"/>
     <addaction name="actionQuick_Replace"/>
     <addaction name="separator"/>
     <addaction name="actionFind_In_File"/>
     <addaction name="actionReplace_In_File"/>
    </widget>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Select_All"/>
    <addaction name="separator"/>
    <addaction name="menuFind_Replace"/>
    <addaction name="actionGo_To_Line"/>
    <addaction name="actionCount_Line"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>&amp;Build</string>
    </property>
    <addaction name="actionBuild"/>
    <addaction name="actionBuild_All"/>
    <addaction name="separator"/>
    <addaction name="actionRun"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionServer_Settings"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionTerminal"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionHide"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionJava_Doc"/>
    <addaction name="separator"/>
    <addaction name="actionTechnical_Support"/>
    <addaction name="actionReport_Bug"/>
    <addaction name="actionSystem_Information"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Superlime"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuBuild"/>
   <addaction name="menu_Tools"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New_File"/>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="action_Undo"/>
   <addaction name="action_Redo"/>
   <addaction name="actionQuick_Find"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="actionServer_Settings"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionZoom_In"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/filenew.png</normaloff>:/icons/Images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/fileopen.png</normaloff>:/icons/Images/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/editundo.png</normaloff>:/icons/Images/editundo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/editredo.png</normaloff>:/icons/Images/editredo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/editcut.png</normaloff>:/icons/Images/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/editcopy.png</normaloff>:/icons/Images/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/editpaste.png</normaloff>:/icons/Images/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Select_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionGo_To_Line">
   <property name="text">
    <string>Go To Line</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/filesave.png</normaloff>:/icons/Images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Exit.png</normaloff>:/icons/Images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCount_Line">
   <property name="text">
    <string>Count Line</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Registry.png</normaloff>:/icons/Images/Registry.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionBuild_All">
   <property name="text">
    <string>Build All</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Play.png</normaloff>:/icons/Images/Play.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionServer_Settings">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Settings.png</normaloff>:/icons/Images/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Server Settings</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionQuick_Find">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Zoom.png</normaloff>:/icons/Images/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>Quick Find</string>
   </property>
  </action>
  <action name="actionQuick_Replace">
   <property name="text">
    <string>Quick Replace</string>
   </property>
  </action>
  <action name="actionFind_In_File">
   <property name="text">
    <string>Find In File</string>
   </property>
  </action>
  <action name="actionReplace_In_File">
   <property name="text">
    <string>Replace In File</string>
   </property>
  </action>
  <action name="actionJava_Doc">
   <property name="text">
    <string>Java Doc</string>
   </property>
  </action>
  <action name="actionTechnical_Support">
   <property name="text">
    <string>Technical Support</string>
   </property>
  </action>
  <action name="actionReport_Bug">
   <property name="text">
    <string>Report Bug</string>
   </property>
  </action>
  <action name="actionSystem_Information">
   <property name="text">
    <string>System Information</string>
   </property>
  </action>
  <action name="actionAbout_Superlime">
   <property name="text">
    <string>About Superlime</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/qt-logo.png</normaloff>:/icons/Images/qt-logo.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/Stop.png</normaloff>:/icons/Images/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionTerminal">
   <property name="text">
    <string>Terminal</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/zoomout.png</normaloff>:/icons/Images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/Images/zoomin.png</normaloff>:/icons/Images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
